lib/SyTest/Homeserver.pm

```diff
@@ -241,7 +241,7 @@ sub configure_logger
 
       loggers => {
          synapse => {
-            level => "INFO"
+            level => "DEBUG"
```

Comment:
I suspect you didn't want to include this?

Reply:
/me grumbles at muscle memory and `-a`

Reply:
```suggestion
            level => "INFO"
```